### PR TITLE
Add Ruby as Dependency to the Docker container

### PR DESCRIPTION
**Pull request by Munsio**
Part of #300

### CODE REVIEW

**Comment by Munsio** [comment]
Having to build ruby from source increases the Docker build time from ~2:30 to >8:00 minutes.

We should check if it is possible to use prebuilt binaries

File: docs/reports/v0.6.0-test/ollama_granite-code_8b-instruct-q8_0/README.md
Comment (bauersimon) [inline docs/reports/v0.6.0-test/ollama_granite-code_8b-instruct-q8_0/README.md:1]: errrhmmm.... not sure this commit belongs to this branch?

File: Dockerfile [hunk 1]
```diff
@@ -20,6 +32,18 @@ COPY --chown=ubuntu:ubuntu ./Makefile ./Makefile
...
 RUN mkdir -p /app/evaluation
 
+# Install Ruby
+RUN mkdir -p /tmp/compile
+RUN wget https://cache.ruby-lang.org/pub/ruby/3.3/ruby-3.3.4.tar.gz && \
+	tar -xf ruby-3.3.4.tar.gz -C /tmp/compile/ && \
+	rm ruby-3.3.4.tar.gz
+WORKDIR /tmp/compile/ruby-3.3.4
+RUN ./configure --prefix /app/.eval-dev-quality/ruby-3.3.4 --disable-install-doc
+RUN	make install
```
Comment (bauersimon) [inline Dockerfile:42]: And that now takes 8 Minutes, right? :see_no_evil: no prebuilt ruby... like ubuntu package? Guess that one is outdated.

Comment (Munsio) [reply Dockerfile:42]: Added the points into their own Issue so we do keep track: https://github.com/symflower/eval-dev-quality/issues/319

**Review by bauersimon — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (Munsio) [reply Dockerfile:42]: Yeah artifacts would be stuff you want to pass from one Job to the next in the CI

**Pull request merged**